### PR TITLE
migrate community search to use ES7

### PR DESCRIPTION
**Pull request by LeoMcA**
@akatsoulas here's where I got with this on Friday, it should all be functional

Some notes:
- It'll conflict with my AAQ PR because I duplicate the `ProductInnerDoc` which I use here too.
- I wonder if there's an elegant abstraction we can create to wire up the signals for us, or if it's easier to just do it manually. I'm thinking a bit ahead to where we might have a couple of layers of denormalisation, e.g. a GroupInnerDoc in a UserInnerDoc in a QuestionDocument. We'd have to add signals handlers to the relevant Groups, Users and Questions. But then creating an abstraction to handle all that is probably more complex than just doing it manually.

### CODE REVIEW

File: kitsune/search/v2/apps.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from django.apps import AppConfig
+
+
+class SearchV2Config(AppConfig):
```
Comment (akatsoulas) [inline kitsune/search/v2/apps.py:4]: Let's move this to the search dir (a level up).

File: kitsune/search/v2/base.py [hunk 2]
```diff
@@ -22,26 +22,33 @@ def prepare(cls, instance):
...
+                # or a list containing an InnerDoc then we're dealing with locales
                 locale = obj.prepare_locale(instance)
                 if locale and value:
                     obj[f] = {locale: value}
-                # We  populate non-locale object fields here
             else:
                 setattr(obj, f, value)
 
-        obj.meta.id = instance.id
+        obj.meta.id = instance.pk
```
Comment (akatsoulas) [inline kitsune/search/v2/base.py:41]: I like the attention to detail :P

**Review by akatsoulas — APPROVED** [review]
r+wc. Let's merge first the AAQ Doc and then this. I am adding an issue to extract/review any InnerDoc that we have throughout our indices.

File: kitsune/search/v2/base.py (hunk 2 above)
Comment (LeoMcA) [reply kitsune/search/v2/base.py:41]: It's needed! `Profile`'s pk is its `user_id` field

**Pull request merged**